botocore/utils.py

```diff
@@ -19,7 +19,7 @@
 import functools
 import weakref
 
-from six import string_types, text_type
+from six import string_types, text_type, binary_type
```

Comment:
Given my changes from https://github.com/boto/botocore/pull/1180, I'll just go ahead and modify the import before merging.